File: latihanB/java/Hardware.java
class Hardware extends Product {
    private String brand;
    private String model;

    Hardware() {
        this.brand = "";
        this.model = "";
    }

    void setBrand(String brand) {
        this.brand = brand;
    }

    String getBrand() {
        return this.brand;
    }

    void setModel(String model) {
        this.model = model;
    }

    String getModel() {
        return this.model;
    }

}

File: latihanB/python/Hardware.py
from Product import *

class Hardware(Product):
    def __init__(self):
        Product.__init__(self)
        self.brand = ""
        self.model = ""

    def setBrand(self, brand):
        self.brand = brand

    def getBrand(self):
        return self.brand

    def setModel(self, model):
        self.model = model

    def getModel(self):
        return self.model